server2/Arcadia.Assistant/Arcadia.Assistant.AppCenterBuilds/Program.cs

```diff
@@ -46,7 +46,7 @@ private static void Main()
                 builder.RegisterServiceLogging(new LoggerSettings(configurationPackage.Settings.Sections["Logging"]));
 
                 using var container = builder.Build();
-                logger = container.TryResolve<ILogger>(out ILogger val) ? val : null;
+                logger = container.TryResolve(out ILogger val) ? val : null;
```

Comment:
You can use `ResolveOptional` to get service or null, if not registered.

Reply:
Done